workbench-core/example/app/src/routes/dynamicAuthorizationRoutes.ts

```diff
@@ -138,13 +138,38 @@ export function setUpDynamicAuthorizationRoutes(router: Router, service: Dynamic
         if (isGroupNotFoundError(error)) {
           throw Boom.notFound(error.message);
         }
+        if (isTooManyRequestsError(error)) {
+          throw Boom.tooManyRequests(error.message);
+        }
+        throw error;
+      }
+    })
+  );
+
+  router.get(
+    '/authorization/groups/:groupId/is-user-assigned/:userId',
+    wrapAsync(async (req: Request, res: Response) => {
+      try {
+        const { data } = await service.isUserAssignedToGroup({
+          authenticatedUser: res.locals.user,
+          userId: req.params.userId,
+          groupId: req.params.groupId
+        });
+        res.status(200).send(data);
+      } catch (error) {
+        if (isUserNotFoundError(error)) {
+          throw Boom.notFound(error.message);
+        }
+        if (isTooManyRequestsError(error)) {
+          throw Boom.tooManyRequests(error.message);
+        }
         throw error;
       }
     })
   );
 
   router.post(
-    '/authorization/identitypermissions',
+    '/authorization/identity-permissions',
```

Comment:
NIT: I would have just done `POST /authorization/permissions`

Reply:
@htztomic ok to change identity permissions base route to just `authorization/permissions`?